backend/src/main/java/ai/verta/modeldb/entities/metadata/LabelsMappingEntity.java

```diff
@@ -86,21 +87,33 @@ public int hashCode() {
     }
   }
 
-  public static String getVersioningCompositeId(VersioningCompositeIdentifier identifier) {
-    return identifier.getRepoId()
-        + "::"
-        + identifier.getCommitHash()
-        + "::"
-        + ModelDBUtils.getLocationWithSlashOperator(identifier.getLocationList());
+  public static String getVersioningCompositeId(
```

Comment:
getVersioningCompositeIdString

just differentiating the two function as Id and Identifier  is confusing

Reply:
Change to `getVersioningCompositeIdString` & `getVersioningCompositeId`